webapp/channels/src/components/autocomplete_selector.tsx

```diff
@@ -108,21 +127,15 @@ export default class AutocompleteSelector extends React.PureComponent {
         let labelContent;
         if (label) {
             labelContent = (
-                <label
-                    className={'control-label ' + labelClassName}
-                >
+                <label className={'control-label ' + labelClassName}>
```

Comment:
This and the next change seem unrelated.

Reply:
Changes reverted